data-service/templates/data_service.yml

```diff
@@ -2,60 +2,68 @@ AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Resources:
 
-  EnvironmentsTable:
+  EnvironmentTable:
     Type: AWS::DynamoDB::Table
     Properties:
      #TODO: change ddb table names to have region prefix passed in as a resource
      TableName: Environments
      AttributeDefinitions:
        - AttributeName: environmentId
          AttributeType: S
-       - AttributeName: environmentVersion
+       - AttributeName: environmentName
+         AttributeType: S
+       - AttributeName: accountId
+         AttributeType: S
+       - AttributeName: cluster
          AttributeType: S
      KeySchema:
        - AttributeName: "environmentId"
          KeyType: HASH
-       - AttributeName: "environmentVersion"
+       - AttributeName: "environmentName"
          KeyType: RANGE
      ProvisionedThroughput:
        ReadCapacityUnits: 15
        WriteCapacityUnits: 15
+     GlobalSecondaryIndexes:
+       -
+         IndexName: "environment-cluster-index"
```

Comment:
Since we're renaming things, perhaps use the same CamelCase naming as the tables themselves?

Reply:
sure, I think this is the default ddb naming scheme but I can switch it to be consistent